src/dune_rules/compilation_context.mli

```diff
@@ -84,7 +85,9 @@ val package : t -> Package.t option
 
 val vimpl : t -> Vimpl.t option
 
-val modes : t -> Mode.Dict.Set.t
+val modes : t -> Lib_mode.Dict.Set.t
+
+val ocaml_modes : t -> Mode.Dict.Set.t
```

Comment:
Do we really need this function? It seems to be a simpler wrapper of `modes`.

Reply:
I guess not. Removed in https://github.com/jchavarri/dune/pull/4/commits/190e2b9285f4b344e4d4bf0fe2c40ab7761d22e7.